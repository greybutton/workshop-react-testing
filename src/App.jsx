import '@babel/polyfill';
import React from 'react';
import ReactDOM from 'react-dom';
import Main from './components/Main';
import cookie from './cookies';

export default (gon) => {
  ReactDOM.render(
    <Main gon={gon} cookie={cookie} />,
    document.getElementById('root'),
  );
};
