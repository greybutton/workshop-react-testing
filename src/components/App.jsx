import React from 'react';
import MyTabs from './Tabs';

const App = () => (
  <div className="container">
    <h1>Hello</h1>
    <MyTabs />
  </div>
);

export default App;
