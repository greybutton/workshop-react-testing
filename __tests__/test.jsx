import React from 'react';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import App from '../src/components/App';

test('example', () => {
  const wrapper = mount(<App />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
