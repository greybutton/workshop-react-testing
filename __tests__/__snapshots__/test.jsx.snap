// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example 1`] = `
<App>
  <div
    className="container"
  >
    <h1>
      Hello
    </h1>
  </div>
</App>
`;
